### ISSUE TITLE
<persName> instead of <name>?

### ISSUE THREAD

**Issue by joeytakeda** [internetshakespeare/ise2-import#3]
The schema currently only accepts `<name>` in `<respStmt>` but the new conversion code uses `<persName>`. I have no preference one way or another, but we should get this in sync.

**Comment by martindholmes** [comment]
One reason for `<name>` is that the responsibility may lie with orgs or other entities not covered by `<persName>`.

**Comment by telic** [comment]
Yes, the current code distinguishes between people and organisations, outputting `<persName>` for people and `<orgName>` for orgs. Since the code could tell the difference, I thought it would be nice to note the difference in the encoding too. @joeytakeda : what was the reason for limiting the content model of `respStmt` in our ODD?

**Comment by joeytakeda** [comment]
IIRC, I was just following the model set by MoEML and a few of the other HCMC projects. I can't think of any good reason not to use `<orgName>` and `<persName>`.
